### PR TITLE
feat: markdown adapter with latex

### PR DESCRIPTION
**Pull request by donteatfriedrice**
[BS-1563](https://linear.app/affine-design/issue/BS-1563/latex-markdown-导入支持)
[BS-1562](https://linear.app/affine-design/issue/BS-1562/latex-markdown-导出支持)

### CODE REVIEW

**Comment by donteatfriedrice** [comment]
* **#8503** <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/8503?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @donteatfriedrice and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/blocks/src/_common/adapters/markdown.ts [hunk 1]
```diff
@@ -996,6 +1036,9 @@ export class MarkdownAdapter extends BaseAdapter<Markdown> {
       case 'list': {
         return [];
       }
+      case 'inlineMath': {
+        return [{ insert: ' ', attributes: { latex: ast.value } }];
```
Comment (L-Sun) [inline packages/blocks/src/_common/adapters/markdown.ts:1040]: remove console.log

**Review by L-Sun — APPROVED** [review]

**Review by Saul-Mirone — APPROVED** [review]
LGTM

**Comment by Saul-Mirone** [comment]
### Merge activity

* **Oct 10, 4:46 AM EDT**: The merge label 'merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=toeverything&repo=blocksuite) once it meets the requirements.
* **Oct 10, 4:46 AM EDT**: A user added this pull request to the [Graphite merge queue](https://app.graphite.dev/merges?org=toeverything&repo=blocksuite).

**Pull request merged**